roles/openshift_manage_node/tasks/main.yml

```diff
@@ -67,11 +67,17 @@
   delegate_to: "{{ openshift_master_host }}"
 
 - name: Label nodes
-  command: >
-    {{ hostvars[openshift_master_host].openshift.common.client_binary }} label --overwrite node {{ openshift.node.nodename }} {{ openshift.node.labels | oo_combine_dict  }}
-    --config={{ openshift_manage_node_kubeconfig }}
-    -n default
-  when: "'nodename' in openshift.node and 'labels' in openshift.node and openshift.node.labels != {}"
+  oc_label:
+    name: "{{ openshift.node.nodename }}"
+    kind: node
+    state: add
+    labels: "{{ openshift.node.kubelet_args.node-labels }}"
```

Comment:
@mtnbikenc so, openshift.node.kubelet_args.node-labels should be a transformation applied to openshift.node.labels in openshift_facts, it was added to try and ensure that if the node is deleted/recreated then it would be recreated with the same node labels.

It was to work around an issue with cloud environments where the node object is deleted if the host is stopped/suspended.